docs/contract-test-collection.md

```diff
@@ -158,13 +158,13 @@ We can send contract address, view method, and parameters as its own "view" to a
 
 That may not sound very useful, but the brilliant part is that the error we receive contains the information we requested! We can not incur a fee for requesting data or waiting for confirmation from the network to call view methods.
 
-Taquito internally contains a list of lambda contracts. Thus, there is no need to deploy a lambda contract if you are using Mainnet, Hangzhounet, or Granadanet. Taquito will detect the current network and use the appropriate lambda contract.
+Taquito internally contains a list of lambda contracts. Thus, there is no need to deploy a lambda contract if you are using Mainnet, Ithacanet, or Hangzhounet. Taquito will detect the current network and use the appropriate lambda contract.
```

Comment:
This part is not relevant anymore, in version 12, we used a RPC endpoint to execute the views instead of a lambda contract

Reply:
removed

Reply:
This sentence is not relevant too anymore: Taquito internally contains a list of lambda contracts. Thus, there is no need to deploy a lambda contract if you are using Mainnet, Ithacanet, or Hangzhounet. Taquito will detect the current network and use the appropriate lambda contract.